Doc/library/sys.monitoring.rst

```diff
@@ -237,11 +237,11 @@ Setting events globally
 
 Events can be controlled globally by modifying the set of events being monitored.
 
-.. function:: get_events(tool_id: int) -> int
+.. function:: get_events(tool_id: int, /) -> int
 
    Returns the ``int`` representing all the active events.
 
-.. function:: set_events(tool_id: int, event_set: int)
+.. function:: set_events(tool_id: int, event_set: int, /)
```

Comment:
You could possibly complete the type annotations here, while you're about it -- other functions like `free_tool_id` in this file have the explicit `-> None` already:

```suggestion
.. function:: set_events(tool_id: int, event_set: int, /) -> None
```